portality/view/oaipmh.py

```diff
@@ -187,7 +188,7 @@ def decode_resumption_token(resumption_token):
     # attempt to parse the resumption token out of base64 encoding and as a json object
     try:
         j = base64.urlsafe_b64decode(str(resumption_token))
-    except TypeError:
+    except (TypeError, binascii.Error):
```

Comment:
It's the same exception raised here and a line above. Can we use one try-catch to keep it concise?

```
try:
  j = base64.urlsafe_b64decode(str(resumption_token))
  d = json.loads(j.decode("utf-8"))   # convert the bytes to str for pre 3.5 compat
except (TypeError, binascii.Error, ValueError):
  raise ResumptionTokenException()
```

Reply:
Good shout. I've added your suggestions to the PR before merge